sdk/ml/azure-ai-ml/tests/dsl/e2etests/test_dsl_pipeline_on_registry.py

```diff
@@ -35,3 +37,105 @@ def sample_pipeline():
         pipeline_job = sample_pipeline()
         assert registry_client.jobs.validate(pipeline_job).passed
         # TODO: add test for pipeline job create with registered component on registry after support is ready on canary
+
+    @pytest.mark.skip(reason="request body still exits when re-record and will raise error "
```

Comment:
let's sync offline on this. I don't quite get here

Reply:
We need to have a task to track this if skip

Reply:
Let's track using the feature item.

Reply:
use this item to track: https://msdata.visualstudio.com/Vienna/_workitems/edit/2045745